packages/block-editor/src/components/rich-text/index.js

```diff
@@ -984,7 +989,6 @@ export class RichText extends Component {
 			return toHTMLString( {
 				value,
 				multilineTag: this.multilineTag,
-				multilineWrapperTags: this.multilineWrapperTags,
```

Comment:
It's passed down to `toTree`. We no longer need the `multilineWrapperTags` context in `toTree` because we no longer need to go through each line format to determine if it's a regular format or a line format. The formats are separated, so we know for sure that they are line "formats", or information.